docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server]]
+= {fleet-server}
+
+beta[]
+
+{fleet-server} is a component used to centrally manage {agent}s. It serves as a
+control plane for updating agent policies, collecting status information, and
+coordinating actions across agents.
+
+//QUESTION: I think it's worth breaking down in detail how data flows between
+// the agent, Fleet Server, Fleet, and the stack, but I might have some details
+// wrong here. Is this correct?
+
+{fleet-server} is the mechanism that connects {agent}s to {fleet} and handles
+all communication between an {agent}, {fleet}, {es}, and {kib}. {fleet-server}:
+
+* Receives status information from connected {agent}s
+* Retrieves agent policies from {fleet} and stores them in {es}
```

Comment:
Fleet (Kibana) only talks to Elasticsearch, Fleet Server only talks to Elasticsearch. There is no connection between Fleet and Fleet-server directly. ES is the communication layer.